ws in controller sample (#22760)
dotnet/AspNetCore.Docs: aspnetcore/fundamentals/websockets.md

A WebSocket request could come in on any URL, but this sample code only accepts requests for `/ws`.

> A similar approach can be taken in an controller method:

Reviewer: ```suggestion
A similar approach can be taken in a controller method:
```

> A similar approach can be taken in an controller method:

Author: a controller method

> To keep the `Startup` clutter-free, a similar approach can be taken in an action method:

Reviewer: ```suggestion
To keep the `Startup` clutter-free, a similar approach can be taken in a controller method:
```

>To keep the `Startup` clutter-free

I don't know if this is needed. It implies that controllers are the only way to remove code from startup which is wrong.

Reviewer 2: ```suggestion
A similar approach can be taken in an controller method:
```

> [!code-csharp[](websockets/samples/6.x/WebSocketsSample/Controllers/WebSocketController.cs?name=WebSocketController)]

Reviewer 2: ```suggestion
[!code-csharp[](websockets/samples/6.x/WebSocketsSample/Controllers/WebSocketController.cs?name=snippet)]
```
